look-and-feel/css/src/Layout/Header/Header.stories.ts

```diff
@@ -1,12 +1,12 @@
 import type { Args, Meta, StoryObj } from "@storybook/html";
-import logo from "../../../common/assets/logo-axa.svg";
-import "./Header.client.scss";
-import "../../../Button/Button.client.scss";
+import logo from "@axa-fr/design-system-look-and-feel-css/dist/common/assets/logo-axa.svg";
```

Comment:
j'ai sorti le repo common/assets/ dans la racine du coup quand j'importe le fichier directement j'ai une erreur eslint
ESLint: Relative import from another package is not allowed. Use `@axa-fr/design-system/common/assets/logo-axa.svg` instead of `../../../../../common/assets/logo-axa.svg`(import/no-relative-packages